graph/history.go

```diff
@@ -8,6 +9,78 @@ import (
 	"github.com/docker/docker/utils"
 )
 
+// History returns the list of all images used to create this image.
+func (graph *Graph) History(img *image.Image) ([]*image.Image, error) {
```

Comment:
Pushed an extra commit which removes `History` and makes updates the WalkHistory passed in function to pass by value. It is only used in two cases and the pointer reference is held is neither. As I pointed out before, `History` was never actually called.